### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by 1337-server**
# Description
- Fixes crash on HandBrake fail - Now logs correctly when raising errors
- Fixes HB trying to rip more tracks than available
- Fixes No failure/error logged in job log - This is currently only logged in the notifications
- Added more small debug msg to MakeMKV output

Fixes # (issue title here)

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Tested on docker running Ubuntu 22.04

- [x] Docker

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

# Changelog:

Include the details of changes made here
- Change 1
- Change 2

# Logs 
RIPMETHOD:
- backup: 
  -  (with SYS_RAWIO enabled)
  -  [WRATH_OF_THE_TITANS_backup.zip](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/11523907/WRATH_OF_THE_TITANS_backup.zip) (with SYS_RAWIO disabled)
- mkv: 
  - (with SYS_RAWIO enabled)
  - [WRATH_OF_THE_TITANS_mkv.zip](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/11523909/WRATH_OF_THE_TITANS_mkv.zip)(with SYS_RAWIO disabled)
- Audio CD:

### CODE REVIEW

**Comment by 1337-server** [comment]
Ready for review.

For some reason I had to zip the logs, I think it might have been the char encoding.

File: scripts/docker/docker_arm_wrapper.sh
Comment (shitwolfymakes) [inline scripts/docker/docker_arm_wrapper.sh:1]: Please update all instances of `/home/arm/logs/arm.log` in this script to reference a variable storing that filepath, to improve maintainability

Comment (1337-server) [reply scripts/docker/docker_arm_wrapper.sh:1]: Done

**Comment by 1337-server** [comment]
Ready for re-review/merge

**Review by shitwolfymakes — APPROVED** [review]

**Pull request merged**